c2rust-transpile/src/translator/mod.rs

```diff
@@ -1394,7 +1372,7 @@ impl<'c> Translation<'c> {
                     use CastKind::*;
                     match cast_kind {
                         IntegralToPointer | FunctionToPointerDecay | PointerToIntegral => {
-                            return true
+                            return true;
```

Comment:
Are all the `;`s from `rustfmt` with default settings?  I thought I had run that on everything already.